src/schema_generators/complete.cljc

```diff
@@ -29,7 +30,10 @@
   schema.spec.variant.VariantSpec
   (completer* [spec s sub-checker generator-opts]
     (let [g (apply generators/generator s generator-opts)]
-      (if (and (class? s) (isa? s clojure.lang.IRecord) (utils/class-schema s))
+      (if #?(:clj (and (class? s) (isa? s clojure.lang.IRecord) (utils/class-schema s))
+             :cljs (and
+                    ;; TODO: more predicates for cljs record schema?
```

Comment:
Since it seems that you can't use `cljs.core/isa?` in the same way as clj for identifying a record, I'm not sure what to do here in cljs. Using only `schema.utils/class-schema` passes tests, but doesn't feel right